#Figures

Figure 1. Plot of merged data from all RILs genotyped using the parent-based SNP set. Each of the B. rapa ten chromosomes are displayed (A01-A10) with counts coverage of each SNP at each physical position on the chromosome in megabases (Mb). The color indicates the relative ratio of coverage between R500 and IMB211 for every SNP. Black is equal coverage, orange is more IMB211 and blue is more R500.

Figure 2. Plot of merged data from all RILs genotyped at population-based SNP set. Each of the B. rapa ten chromosomes are displayed (A01-A10) with counts coverage of each SNP at each physical position on the chromosome in megabases (Mb). The color indicates the relative ratio of coverage between R500 and IMB211 for every SNP. Black is equal coverage, orange is more IMB211 and blue is more R500.

Table 1. SNP counts at different steps of the SNP discovery pipeline. The percentage of SNPs located on chromosomes or scaffolds remaining after each step are shown in parentheses. The first percentage is relative to the initial set of SNPs and the second percentage is relative to the set of SNPs from the previous step.

Table 2. Incorporated scaffolds represent a disproportionately high amount of scaffold sequence. Percentages of scaffold subset counts and total lengths relative to the set of all scaffolds are shown in parentheses.

Figure 3. (A) Number of SNPs per scaffold. (B) Density distributions of scaffold sizes. Newly incorporated scaffolds are shown in green and unincorporated scaffolds are shown in gray.

Figure 4. Composite population genotype map with the position in physical distance across all ten chromosomes. Each RIL is represented as a single row displaying the genomic region inherited from IMB211 (Orange) or R500 (Blue). Small heterogeneous regions are represented in black. 

Figure S1. Representative asymmetric binary distance analysis plot for a single marker located on A01. Markers with X% correlation are indicated in red. Misplaced markers are circled in blue. 

Figure 5. Asymmetric binary distance plots for each marker compared against every other marker before (A) and after (B) marker rearrangement. Dark blue indicates high correlation while white indicates no genetic correlation. 

Figure 6. Old and new genetic map comparisons. Genetic markers for each chromosome are displayed in centimorgan distance (cM) for the old (A) and new (B) genetic maps. Comparison of likelihood odds scores for flowering time QTL using the old (C) and new (D) genetic maps.

Figure S2 Physical position versus genetic position of each marker for genome version 1.5 with spline fits for converting between genetic and physical distances.


