Introduction
The Brassica genus provides up to 10% of human calories and is important for providing micronutrients in human diets throughout Asia (book ref, fao/stat/). Recently sequenced species within this genus include Brassica napus, Brassica rapa, and Brassica oleracea (Brassica rapa Genome Sequencing Project Consortium et al. 2011; Liu et al. 2014)., Napus ref.). Brassica rapa is a physiologically and morphologically diverse diploid species has 87% gene exon similarity to the model plant Arabidopsis thaliana (Cheng et al. 2013). This makes Brassica rapa a great candidate for comparing and translating knowledge of biological processes into a crop species from Arabidopsis. For example, using homologous Arabidopsis gene information to infer the action of B. rapa genes in flowering time (ref), leaf development(ref), seed yield and oil content (ref). All of these important traits contribute to our understanding of plant growth in agricultural settings and are made possible by a reference genome sequence (refs) and improved gene annotation (refs, devesitty). 

Brassica rapa has a genome size of 283.4 Gb spread over 10 chromosomes A01-A10 (Wang et al. 2011).

The genome consortium has a public release at version 1.5 that is a significant improvement over version 1 (“Brassica Database | Brassica genome sequence”). 

However, there are still many large scaffolds that have not been placed due to various problems with assembly.

These scaffolds equate to X% of the remaining sequence in the genome. An increasingly common way to place scaffolds and contigs is to use recombination probabilities between molecular markers that occur on those scaffolds so they can be placed in relative positions if general genome assembly fails.

The most time consuming part of this step is making a genetic mapping population and completing the sequencing on the progeny (refs).

We chose a heavily used recombinant inbred line population developed by .







