Introduction
The Brassica genus provides up to 10% of human calories and is important for providing micronutrients in human diets throughout Asia (book ref, fao/stat/). Recently sequenced species within this genus include Brassica napus, Brassica rapa, and Brassica oleracea (Brassica rapa Genome Sequencing Project Consortium et al. 2011; Liu et al. 2014)., Napus ref.). Brassica rapa is a physiologically and morphologically diverse diploid species has 87% gene exon similarity to the model plant Arabidopsis thaliana (Cheng et al. 2013). This makes Brassica rapa a great candidate for comparing and translating knowledge of biological processes into a crop species from Arabidopsis. For example, using homologous Arabidopsis gene information to infer the action of B. rapa genes in flowering time (ref), leaf development(ref), seed yield and oil content (ref). All of these important traits contribute to our understanding of plant growth in agricultural settings and are made possible by a reference genome sequence (refs) and gene annotation information (refs, devesitty). 

Brassica rapa has a genome size of 283.4 Gb spread over 10 chromosomes A01-A10 (Wang et al. 2011). Although the current genome is diploid, there are three ancient subgenomes called Least Fractionated (LF), most fractionated one (MF1), and most fractionated two (MF2). These three subgenomes share many homologous genes and contigous regions adding the difficulty in assembling chromosomes in the correct order. For example, there are still many gene containing genomic scaffolds in version 1.5 of the genome (brassica db ref)that have not been placed on a chromosome due to lack of information as to where they belong. These scaffolds equate to ~X% of the remaining sequence left to be placed in the genome. 

An increasingly common way to place scaffolds is to use molecular marker recombination information between scaffolds and flanking genomic sequence (Hahn et al. 2013). In fact, this approach was used for the original Brassica rapa genome assembly, but there were still missing molecular markers on the unplaced scaffolds. The most time consuming part of this step is making a genetic mapping population and sequencing the progeny for genotype information (refs). There is also no guarantee that there will be recombination events and molecular markers around the scaffolds in the mapping population.

For this study we leveraged an existing recombinant inbred line population of Brassica rapa (BraIIRRI, ref). This population has been used extensively for QTL mapping of physiological, developmental, and evolutionarily important traits (refs). Recently, we completed deep RNA-sequencing of the parents of the Bra-IRRI population providing a large SNP set and improved gene annotation information (Devisetty et al. 2014). We now extend these methods to 124 genotypes in the population for SNP discovery, the creation of a saturated genetic map, and fixing genome misassemblies and placing scaffolds. 








